src/main/java/com/welovecoding/nbeditorconfig/options/EditorConfigOptionsPanel.java

```diff
@@ -17,6 +17,7 @@
    * WARNING: Do NOT modify this code. The content of this method is always
    * regenerated by the Form Editor.
    */
+  @SuppressWarnings("unchecked")
```

Comment:
I think suppressing warnings is not a good practice. We should find the root cause of this issue. 😃